package.json

```diff
@@ -2,9 +2,15 @@
   "name": "platform-chaos",
   "version": "1.0.0",
   "description": "A node sdk for building services capable of injecting chaos into PaaS offerings",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "mocha"
+    "lint": "npx tslint -c tslint.json 'src/**/*.ts'",
+    "pretest": "npm run cleanup && npm run compile",
+    "test": "npm run lint && npm run test-only",
+    "posttest": "npm run cleanup",
```

Comment:
for instance, this seems unneeded

Reply:
I added this because when you run `npm run test` it compiles then runs the test suite. Considering that the user is attempting to run unit tests there shouldn't be side effects like a leftover `dist/` directory.

Reply:
i don't think i agree, as test shouldn't compile code at all, so imo pretest and posttest should both be dropped, and leave dist management to `npm run compile`